### ISSUE TITLE
Google Chrome security changes

### ISSUE THREAD

**Issue by jduckles** [carpentries/lesson-example#133]
TL;DR - The use of `<input>` on the lesson pages to enable a Google search is causing problems under some (non SSL) configurations due to an upcoming change pushed out to Chrome

This has to do with the following warning from Google:
> Starting October 2017, Chrome (version 62) will show a "NOT SECURE" warning when users enter text in a form > on an HTTP page, and for all HTTP pages in Incognito mode.

While most github pages are hosted at `https://<orgname>.github.io/<reponame>`, if the GitHub account has been configured to use a custom domain, SSL **cannot** be configured for pages hosted by GitHub. Data Carpentry has a custom domain configuration for their lessons, so will encounter this problem.

Possible solutions include, but may not be limited to:
​1) remove search box from DC lessons (advantage: simple, disadvantage: loss of search and still not HTTPS)
​2) remove search box from upstream template (advantage: will work for custom domain configs with no warnings, disadvantage: loss of search and still not HTTPS)
​3) revert to using datacarpentry.github.io form URLs for lessons (advantage: keep search and get HTTPS, disadvantage: Old URLS will need redirects)
​4) host lessons elsewhere with a proper SSL configuration (not hard, but not quite as easy or simple as the current config) related to #126

**Comment by fmichonneau** [comment]
all our websites are now served over SSL
